libs/console/full/include/console/console.h

```diff
@@ -29,7 +29,7 @@ void console_write(const char *str, int cnt);
 int console_read(char *str, int cnt);
 void console_blocking_mode(void);
 void console_echo(int on);
-
+void console_print_prompt(void);
```

Comment:
We should keep the API in the other console package exactly the same as here. I.e. add a matching stub to console/stub/include/console.h.